web/modules/custom/collection/collection.module

```diff
@@ -109,7 +109,7 @@ function collection_rdf_entity_access(EntityInterface $entity, $operation, Accou
 
     case 'edit':
       $allowed_transitions = $entity->field_ar_state->first()->getTransitions();
-      return AccessResult::forbiddenIf(empty($allowed_transitions));
+      return empty($allowed_transitions) ? AccessResult::forbidden() : AccessResult::allowed();
```

Comment:
FYI, we faced the same issue with the node workflows and we ended up having this way of checking.